gefserver/pier/pier.go

```diff
@@ -380,10 +437,13 @@ func (p *Pier) runJob(job *db.Job, service db.Service, inputPID string, limits d
 	p.updateJobDurationTime(*job)
 }
 
-// RemoveVolumeInUse removes a volume that may seem to be in use
-func (p *Pier) WaitAndRemoveVolume(id dckr.VolumeID) error {
+func (p *Pier) waitAndRemoveVolume(connectionID db.ConnectionID, id dckr.VolumeID) error {
```

Comment:
Comment to the function is missing

Reply:
The internally used (unexported) functions don't need comments, go vet does not complain about this.

Reply:
ok, I was not sure